docs/sandbox/MapboxVectorTilesApi.md

```diff
@@ -40,40 +43,63 @@ The feature must be configured in `router-config.json` as follows
       "minZoom": 12,
       "cacheMaxSeconds": 600
     },
+    // all rental places: stations and free-floating vehicles
     {
       "name": "citybikes",
-      "type": "VehicleRental", // all rental places: stations and free-floating vehicles
+      "type": "VehicleRental",
       "mapper": "Digitransit",
       "maxZoom": 20,
       "minZoom": 14,
       "cacheMaxSeconds": 60,
       "expansionFactor": 0.25
     },
+    // just free-floating vehicles
     {
       "name": "rentalVehicles",
-      "type": "VehicleRentalVehicle", // just free-floating vehicles
-      "mapper": "Digitransit",
+      "type": "VehicleRentalVehicle",
+      "mapper": "DigitransitRealtime",
       "maxZoom": 20,
       "minZoom": 14,
       "cacheMaxSeconds": 60
     },
+    // just rental stations
     {
       "name": "rentalStations",
-      "type": "VehicleRentalStation", // just rental stations
+      "type": "VehicleRentalStation",
       "mapper": "Digitransit",
       "maxZoom": 20,
       "minZoom": 14,
       "cacheMaxSeconds": 600
     },
+    // Contains just stations and realtime information for them
+    {
+      "name": "rentalStations",
```

Comment:
We have the same layer name several times in the example. Would this lead to a runtime error during startup?

If you want to keep the same name, can you make it clear in the comment that you can use either/or?

Reply:
Thanks for pointing this out. Forgot to changes the name for this. For vehicle parking, I added a comment that the other layer was for backwards compatibility reasons and has the same name